kbi/0001/index.rst

```diff
@@ -0,0 +1,187 @@
+.. index::
+   single: datalad; clone
+   single: config; safe.directory
+.. highlight:: text
+
+KBI0001: Security considerations for accessing shared datasets on multi-user machines
+=====================================================================================
+
+:authors: Michał Szczepanik <m.szczepanik@fz-juelich.de>
+:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/1
+:keywords: clone, shared, safe.directory
+
+Overview
+--------
+
+Some users who want to clone Git repositories (DataLad datasets) from
+locations on a machine that are owned by other users report receiving
+an error message containing the following statement::
+
+   fatal: unsafe repository ('/path/to/repository' is owned by someone
+   else)
+
+or::
+
+   Git refuses to operate in this repository, probably because it is
+   owned by someone else.
+
+The error message comes from Git and also contains the following
+suggestion::
+
+   To add an exception for this directory, call:
+
+	git config --global --add safe.directory /path/to/repository
+
+This technical note provides background information about the observed
+behavior, and guidance on possible solutions.
+
```

Comment:
```suggestion


Background
----------

```

What comes next is already background, and I think there is no need for "further background" below.